src/backend/distributed/sql/citus--8.3-1--9.0-1.sql

```diff
@@ -33,6 +33,11 @@ CREATE TABLE citus.pg_dist_object (
     objid oid NOT NULL,
     objsubid integer NOT NULL,
 
+    -- fields that are only valid for distributed
+    -- functions/procedures
+    distribution_argument_index int,
+    colocationid int,
```

Comment:
It would make a bit more sense for these to be at the end, since we might add more of these class-specific fields later.